### ISSUE TITLE
Crash during game load on Apple Silicon

### ISSUE THREAD

**Issue by mkaito** [Kopernicus/Kopernicus#575]
I've been able to reproduce this with any mod that depends on Kopernicus. I've tried AVP and Outer Planets Mod. The game will crash and get stuck while lldb tries to figure out what happens, then that also gets stuck and I have to SIGKILL lldb. Player.log is fairly big, but attached below.

Let me know if I can do anything else to help the debugging.

[Player.log](https://github.com/Kopernicus/Kopernicus/files/10949333/Player.log)

**Comment by R-T-B** [comment]
This is, if I had to fathom a guess, caused most likely by our use of Harmony to patch the stock dlls?

There are very old builds that don't depend on harmony that may work if that is the case.  But if it's really an issue with harmony, I can't fix it from my side as we need that dependency too strongly, sadly.

I'll touch bases with the KSPHarmony framework authors to see if we can't sort out what's going on, and if they can't fix it.

In the meantime, you may find the last release that doesn't use harmony may work for you.  That is: https://github.com/Kopernicus/Kopernicus/releases/tag/release-132

Most of the bugfixes since then were performance onces, so mods should work with it, just way slower.  Regards.

**Comment by R-T-B** [comment]
Linking this upstream issue to monitor:

https://github.com/pardeike/Harmony/issues/424

**Comment by mkaito** [comment]
That's interesting. I see Rosetta being mentioned as a workaround, but KSP1 never got an ARM build, and so is already running through Rosetta.

**Comment by R-T-B** [comment]
Yeah, I can only advise old builds until they figure out what's what there.

One trick you may try is granting "Executable" permission to the Harmony and Kopernicus dlls.  It's a longshot but it may just let Rosetta run them.
